app/views/admin/product_import/_entries_table.html.haml

```diff
@@ -0,0 +1,12 @@
+- if entries && entries.count > 0 #&& entries.values.first && entries.values.first[:entry]
+  %div.table-wrap
```

Comment:
There isnt actually a need for `%div` here and in other places. Its assumed to be the default if just a class or ID is used